conjure/controllers/summary/tui.py

```diff
@@ -1,15 +1,14 @@
 from . import common
 from conjure import utils
-import sys
 
 
 class SummaryController:
     def finish(self):
-        sys.exit(0)
+        return
```

Comment:
if all we're doing is returning here, we could just not have finish() and not call it in render...